wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/UIAlertController+ConversationGuestOptions.swift

```diff
@@ -33,14 +33,27 @@ extension UIAlertController {
         return controller
     }
 
-    static func confirmRemovingGuests(_ completion: @escaping (Bool) -> Void) -> UIAlertController {
+    static func confirmRemovingGuests(
+        _ completion: @escaping (Bool) -> Void
+    ) -> UIAlertController {
         return confirmController(
             title: GuestRoom.RemoveGuests.message,
             confirmTitle: GuestRoom.RemoveGuests.action,
             completion: completion
         )
     }
 
+    static func confirmGuestLinkCreationWithOptionalPassword(
+        _ completion: @escaping (Bool) -> Void
+    ) -> UIAlertController {
+        return confirmControllerWithTwoOptions(
+            optionOne: GuestRoom.Create.LinkWithPassword.action,
+            optionTwo: GuestRoom.Create.LinkWithoutPassword.action,
+            completion: completion
+        )
+
+    }
```

Comment:
It's not clear what the `Bool` represents... you could, for instance, make it clearer by returning a type `GuestLinkType.secure` or `GuestLinkType.normal`.

Reply:
@johnxnguyen I've addressed this, thanks!